peregrine/resources/submission/graphql/node.py

```diff
@@ -8,7 +8,7 @@
 """
 
 
-from flask import current_app as capp
+from flask import current_app as capp, g as fg
```

Comment:
I know there's this `capp` and `fg` convention floating around from old code, but could you use
```
import flask

flask.g
```
instead for new code?

Reply:
Yeah sure thing!